icu4c/source/tools/icu-file-utf8-check.py

```diff
@@ -61,16 +67,20 @@ def check_file(file_name, is_source):
             bytes.decode("UTF-8")
         except UnicodeDecodeError:
             print("Error: %s is a source code file but contains non-utf-8 bytes." % file_name)
-    
+            rc = 1
+
     if bytes[0] == 0xef:
         if not (file_name.endswith(".txt") or file_name.endswith(".sln")
-                    or file_name.endswith(".targets")
+                    or file_name.endswith(".targets") or file_name.endswith(".md")
```

Comment:
Are the markdown files all UTF-8 with BOM? I wonder if we might want to remove the BOM on them too...

Reply:
Hopefully the Markdown files do *not* start with a BOM.

Reply:
Right -- I don't know if we want to start skipping the `.md` files.

Reply:
The script flagged one MD file:
icu4c/source/test/testdata/break_rules/README.md
I thought it's close enough to a .txt file and added .md to the filter. Apparently MD files are not that close after all.